hedera-node/src/test/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogicTest.java

```diff
@@ -69,6 +86,81 @@ public void hasCorrectApplicability() {
         assertFalse(subject.applicability().test(TransactionBody.getDefaultInstance()));
     }
 
+    @Test
+    public void followsHappyPath() {
```

Comment:
This test verifies that the correct methods are invoked, but does not verify that the methods were invoked with the correct parameters. I think that it would be even better if we replace the `any()` arguments in `given` and `verify` statements to actually verify that all methods were invoked with the correct parameters